docs/architecture/adr-101-data-companion-pull-api.md

```diff
@@ -0,0 +1,403 @@
+# ADR 101: Data Companion Pull API
+
+## Changelog
+
+- 2023-05-03: Update based on synchronous feedback from team (@thanethomson)
+- 2023-04-04: Update based on review feedback (@thanethomson)
+- 2023-02-28: Renumber from 084 to 101 (@thanethomson)
+- 2022-12-18: First draft (@thanethomson)
+
+## Status
+
+Accepted
+
+## Context
+
+See the [context for ADR-100][adr-100-context].
```

Comment:
```suggestion
See the [context for the Data Companion Push API (ADR-100)][adr-100-context].
```

Reply:
I don't think we need to repeat this description of ADR-100 thorough the document, but at least here, the first time it is used, it would be good for the reader.